api/predict.py

```diff
@@ -2,6 +2,7 @@
 from maxfw.core import MAX_API, PredictAPI
 from flask_restplus import fields
 from flask import abort
+import copy
```

Comment:
Is `copy` used?

Reply:
I will remove it, it was flagged by flake8 as well.